src/Microsoft.PowerShell.Security/security/CatalogCommands.cs

```diff
@@ -114,10 +115,13 @@ protected override void ProcessRecord()
                 catalogFilePath = SessionState.Path.GetUnresolvedProviderPathFromPSPath(catalogFilePath);
             }
 
-            if (ShouldProcess(catalogFilePath))
+            string action = StringUtil.Format(Modules.CreatingModuleManifestFile, catalogFilePath);
```

Comment:
I noticed here you are using the 
```
Modules.CreatingModuleManifestFile
```
property which comes from the file:

```
PowerShell/src/System.Management.Automation/resources/Modules.resx
```

Rather than using that one, you should add a _new_ entry in that `resx` file called like `CreatingFileCatalog`

and use it instead.



NOTE: In order to rebuild the `resx` files... instead of running 

`Start-PSBuild`
you run
`Start-PSBuild -Clean`

Reply:
Actually @TylerLeonhardt if all you're doing is regenerating your resx files you want to `Start-PSBuild -ResGen`

But yeah clean build is easy and covers it for ya. 😄